packages/eslint-config-eslint/default.yml

```
@@ -13,7 +13,7 @@ rules:
     camelcase: "error"
     callback-return: ["error", ["cb", "callback", "next"]]
     class-methods-use-this: "error"
-    comma-dangle: "error"
+    comma-dangle: ["error", {functions: "never"}]
```

Nit: Could we have spacing inside the brackets, to emulate our ESLint settings?

```suggestion
    comma-dangle: ["error", { functions: "never" }]
```